### ISSUE TITLE
Clean up /dor/export folder for accessioned items

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#1878]
**Describe the bug**
After items are accessioned, a folder is left in `/dor/export` with a name that follows this pattern:

`{druid}_v{version}`

For example, there's currently a folder at this path `/dor/export/zz601dq6831_v1` 

Inside this folder is a BagIt structure with symlinks to files in the druid tree for the item in `/dor/workspace` for all the files in the `content` and `metadata` folders. When that corresponding druid tree in `/dor/workspace` is cleaned up after accessioning, this leaves behind broken symlinks in the `dor/export` folder.

**User Impact**
There isn't a noticeable impact on most users but over time the /dor/export folder will accumulate enough data to be a problem. Fortunately, broken symlinks don't take up much space.

This also makes it difficult to monitor space in the `/dor` workspace since it's not easy to distinguish folders that have broken symlinks from folders that have content that takes up more space.

**Expected behavior**
I am not sure what the correct cleanup state is supposed to be, since I'm not sure if there is supposed to be something left behind in `/dor/export`. But I don't think the folder is supposed to contain broken symlinks.

Note that `/dor/export` is used to create the bag that is sent from the `/dor` workspace to the preservation disk. So that must be how the BagIt structure is created in the first place.

**Comment by jcoyne** [comment]
this is `Settings.sdr.local_export_home` in common-accessioning.

**Comment by jcoyne** [comment]
Related to https://github.com/sul-dlss/common-accessioning/issues/321

**Comment by jcoyne** [comment]
Existing code that touches that is here:
```
https://github.com/sul-dlss/dor-services-app/blob/master/app/controllers/workspaces_controller.rb#L30-L31
```

**Comment by ndushay** [comment]
is this done?  If so, please close the ticket.  If not - can it be attached to a PR (near bottom of PR in zenhub view)

**Comment by andrewjbtw** [comment]
This will still need a review on stage.

**Comment by andrewjbtw** [comment]
Updating for 2022: this remains a problem. I have a vague memory of a PR that attempted to fix it but either we decided not to take that approach or it did not ultimately fix the problem.

I remember there was a question of whether it is necessary to give the "export" folder a suffix like "_v3" to represent a version that successfully completed accessioning. I think we should continue to do that until we've fixed this cleanup issue. It's hard to follow what's happening on the filesystem while the system is leaving behind these legacy folders. Currently the "_v#" suffix is protecting the system from accidentally deleting files that are still being processed.
